src/Symfony/Component/Console/Tests/Helper/HelperTest.php

```diff
@@ -15,6 +15,21 @@
 
 class HelperTest extends \PHPUnit_Framework_TestCase
 {
+    public function formatMemoryProvider()
+    {
+        return array(
+            array(0,          '0 B'),
```

Comment:
I know there are many implicit rules that aren't yet declared @SpacePossum, that's why I'm asking. The reasons I have in mind are the very same that moved us to unalign `=` and `=>` in the past (#14438), given the code maintainability turns harder when any update to this "structures" is required.